languagetool-core/src/main/java/org/languagetool/tagging/BaseTagger.java

```diff
@@ -114,24 +116,51 @@ protected WordTagger getWordTagger() {
   private WordTagger initWordTagger(boolean internTags) {
     MorfologikTagger morfologikTagger = new MorfologikTagger(dictionary, internTags);
     try {
-      String manualRemovalFileName = getManualRemovalsFileName();
       ManualTagger removalTagger = null;
-      if (manualRemovalFileName != null) {
-        try (InputStream stream = JLanguageTool.getDataBroker().getFromResourceDirAsStream(manualRemovalFileName)) {
+      InputStream stream = null;
+      try {
+        for (String file : getManualRemovalsFileNames()) {
+          for (URL url : JLanguageTool.getDataBroker().getFromResourceDirAsUrls(file)) {
+            if (stream == null) {
+              stream = url.openStream();
+            } else {
+              stream = new SequenceInputStream(stream, url.openStream());
```

Comment:
Does this really work for more than two items in the `for` loop?

Reply:
Why this shouldn't work? [Here](https://stackoverflow.com/questions/14295099/how-to-chain-multiple-different-inputstreams-into-one-inputstream) is a good discription for such solution.

Reply:
Ah, okay!